src/uct/api/tl.h

```diff
@@ -32,181 +287,125 @@ typedef struct uct_iface_ops {
 
     /* endpoint - put */
 
-    ucs_status_t (*ep_put_short)(uct_ep_h ep, const void *buffer, unsigned length,
-                                 uint64_t remote_addr, uct_rkey_t rkey);
+    uct_ep_put_short_func_t             ep_put_short;
```

Comment:
maybe we should remove all space lines, and the comments like `/* endpoint - put */`?

Reply:
as per offline discussion: left space line between sections only, left comments to simplify reading